### ISSUE TITLE
File in-out node encoding selection menu is not displayed with Firefox. 

### ISSUE THREAD

**Issue by utaani** [node-red/node-red#2086]
### What are the steps to reproduce?
Place file-out  node, 

### What happens?
The encoding selection menu is not displayed with Firefox. 
JA encode
![firefox](https://user-images.githubusercontent.com/17796/54244333-8bc2da00-456f-11e9-9fc7-3b3506a98718.png)

EN encode
![firefox-en](https://user-images.githubusercontent.com/17796/54244451-0855b880-4570-11e9-9fde-b07180e47558.png)

### What do you expect to happen?
It is displayed correctly with Chrome.
![chrome](https://user-images.githubusercontent.com/17796/54244365-b01eb680-456f-11e9-9e35-f983c1cde4be.png)

### Please tell us about your environment:

- [ ] Node-RED version: 0.20.0
- [ ] node.js version: 8.15.1
- [ ] npm version: 6.4.1
- [ ] Platform/OS: raspbian stretch
- [ ] Browser: Firefox 65.0.2 ja/en

**Comment by utaani** [comment]
The part of menu include nested <optgroup>, its not fit with HTML5 spec.
c.f. https://www.fxsitecompat.com/en-CA/docs/2015/nested-optgroups-are-no-longer-allowed/
```
<select type="text" id="node-input-encoding" style="width: 250px;">
    <option value="none" label="デフォルト"></option>
    <optgroup label="UNICODE">
        <option value="utf-16be" label="utf-16be"></option>
    </optgroup>
    <optgroup label="日本">
        <option value="Shift_JIS" label="Shift_JIS"></option>
        <option value="Windows-31j" label="Windows-31j"></option>
        <option value="Windows932" label="Windows932"></option>
        <option value="EUC-JP" label="EUC-JP"></option>
    </optgroup>
...
```
Does this matter with this issue?

**Comment by HiroyasuNishiyama** [comment]
The problem seems to be caused by a bug of Firefox on displaying *label* attribute of *option* tag as discussed [here](https://github.com/praeclarum/Ooui/issues/103).
Will make PR for workaround of this issue.

**Comment by knolleary** [comment]
Fixed in master - will be in 0.20.1
